src/components/PlayList/playlist.css

```diff
@@ -0,0 +1,73 @@
+:root{
+    --block-background:rgba(86, 177, 63, 0.8);
+}
+.video-thumbnail {
+    max-height: 14rem;
+}
+
+.video-card {
+    position: relative;
+}
+
+.half-block {
+    position: absolute;
+    display: flex;
+    flex-direction: column-reverse;
+    justify-content: center;
+    align-items: center;
+    width: 45%;
+    height: 14rem;
+    background-color: var(--block-background);
+    top: 0;
+    right: 0;
+}
+
+.delete {
+    position: absolute;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 4rem;
+    height: 4rem;
```

Comment:
Suggestion: you could combine common properties like `display `and `align-items` to avoid repetition.

for-example:
.half-block, .delete {
    display: flex;
    justify-content: center;
    align-items: center;
    position: absolute;
    top:0;
    right:0;
.....
}

Reply:
Thank you for all your suggestions...